### PR TITLE
Game logic

### PR DESCRIPTION
**Pull request by elpotatoslayer**
- Make `Board` class display and handle the board logic
- Make a `Game` class handle to the core logic of the game.
- Make a `Player` class assign players with a `mark.`

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__Your project is amazing. You look like the type that pays attention to details.
I particularly like the flow of your application. Overall, this project shows that you've dedicated a lot of time to it. Please, keep it up! :+1: 
You just have to make some changes before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] At the moment, your application cannot tell if there was a draw after players move. Kindly look into this.
- [ ] You did not use Ruby attribute accessors (attr_reader/attr_writer or attr_accessor) in your classes. Kindly review your classes and use the proper accessor for your instance variables. 
- Note: You should use attr_accessor only if you really need it and must be aware that by using it, you allow writing that property from outside the class. This means that a variable initialized with attr_accessor can be set to anything both within and outside the class. Check [this article](https://www.rubyguides.com/2018/11/attr_accessor/) for details.



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by mke2111 — APPROVED** [review]
# **STATUS: APPROVED**

### Congratulations Team. I applaud you for your efforts towards this milestone. The game works perfectly fine. Keep the motivation up.

## **OPTIONAL CHANGES:**

- [ ] It would be nice if the same players have an option of playing again, in case they are not tired of Tic-tac_toes.

**Cheers!!** 
image

**Pull request merged**
